projects/i18n/switch/language-switcher.options.ts

```diff
@@ -0,0 +1,16 @@
+import {InjectionToken} from '@angular/core';
+import {TuiLanguageName} from '@taiga-ui/i18n/interfaces';
+
+export type TuiLanguageLoader = (lang: TuiLanguageName) => Promise<unknown>;
+
+export interface TuiLanguageStorage extends Storage {
```

Comment:
Shouldn't this be in interfaces folder? Seems unrelated to options.

Reply:
fixed